pkgs/applications/networking/remote/tlclient/default.nix

```
@@ -0,0 +1,72 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, copyDesktopItems
+, makeDesktopItem
+, alsa-lib
+, libX11
+, pcsclite
+}:
+let
+  tlclient-png = fetchurl {
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/tlclient.png?h=tlclient&id=fd126edd54a012caa8c9ce47ab8d7fb25e982e74";
+    sha256 = "12i0wc4a8r69kwcdj0kjidwvmwhpb8jcwsqxw9dl5vm1gdqxyz5v";
+    name = "tlclient-png";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "tlclient";
+  version = "4.16.0-3389";
+
+  src = fetchurl {
+    # x86_64 arch tarball
+    url = "https://www.cendio.com/downloads/clients/tl-${version}-client-linux-dynamic-x86_64.tar.gz";
+    sha256 = "ea3a92186cbaf71cc4ab692b3baeadd94b3836242e58705a38fe3683f4f18691";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook copyDesktopItems ];
+  propagatedBuildInputs = [ alsa-lib libX11 pcsclite ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 "lib/tlclient/EULA.txt" "$out/share/licenses/tlclient/EULA.txt"
+    install -m644 "lib/tlclient/open_source_licenses.txt" "$out/share/licenses/tlclient/open_source_licenses.txt"
+    cp -R lib "$out/"
+
+    install -Dm644 "etc/tlclient.conf" "$out/etc/tlclient.conf"
+    install -Dm755 "bin/tlclient" "$out/bin/tlclient"
+    install -Dm755 "bin/tlclient-openconf" "$out/bin/tlclient-openconf"
+
+    install -Dm644 "${tlclient-png}" "$out/share/tlclient/tlclient.png"
+
+    runHook postInstall
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      desktopName = "ThinLinc Client";
+      name = "tlclient";
+      exec = "tlclient";
+      icon = "tlclient";
+      comment = meta.description;
+      type = "Application";
+      categories = [ "Network" "RemoteAccess" ];
+    })
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
tracking issue: #292468